snippets/nordic-log-stm/boards/nrf54h20_cpuapp.overlay

```diff
@@ -3,6 +3,25 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+/ {
+	reserved-memory {
+		etr_buf_ram0x_region: memory@2f0be000 {
```

Comment:
Now that the buffer location is flexible, we should move it to `cpuapp_ram0x_region` instead of maintaining this disjoint region, re: https://github.com/zephyrproject-rtos/zephyr/pull/72273#discussion_r1791261444

Reply:
I did that initially but that means that either `cpurad_ram0x_region` needs to move down by 4K or `cpuapp_data` in `cpuapp_ram0x_region` must be reduced by 4K and figured both aren't that great. 
We are going to have a coalescing PR for memory regions anyway after dropping GenB support, maybe it could be done there?

Reply:
Not meaning to kick the ball just further to the next person, I can start having a look at the memory map given recent priority changes

Reply:
The snippet should shrink the cpuapp's SRAM in cpu_app_ram0x_region and place etr_buf_ram0x in there